chart/kepler/templates/model-server-deployment.yaml

```diff
@@ -0,0 +1,74 @@
+{{- if .Values.modelServer.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "modelServer.fullname" . }}
+  labels:
+    {{- include "modelServer.labels" . | nindent 4 }}
+  {{- with .Values.modelServer.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.modelServer.replicas }}
+  selector:
+    matchLabels:
+      {{- include "modelServer.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.modelServer.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "modelServer.selectorLabels" . | nindent 8 }}
+        {{- with .Values.modelServer.podLabels }}
+        {{- . | toYaml | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: server-api
+        args:
+        - model-server
+        image: "{{ .Values.modelServer.image.repository }}:{{ .Values.modelServer.image.tag }}"
+        imagePullPolicy: {{ .Values.modelServer.image.pullPolicy }}
+        ports:
+        - containerPort: 8100
+          name: http
+          protocol: TCP
```

Comment:
What about `startupProbe`/`readinessProbe`/`livenessProbe`?

Reply:
I added a `readinessProbe`. For the `livenessProbe` I commented above whether we really want it, that questions also applies here. If we want to add a `livenessProbe`, I'll also add the `startupProbe` with it.

Reply:
okay, I added it.